src/qml/qgismobileapp.qml

```diff
@@ -1135,8 +1135,22 @@ ApplicationWindow {
       leftPadding: 10
 
       onTriggered: {
-        printMenu.popup( mainMenu.x, mainMenu.y + printItem.y )
-        highlighted = false
+        if (layoutListInstantiator.model.rowCount() > 1)
+        {
+          printMenu.popup( mainMenu.x, mainMenu.y + printItem.y )
+          highlighted = false
+        }
+        else
+        {
+          mainMenu.close();
+          displayToast( qsTr( 'Printing to PDF') )
+          printMenu.printTimer.interval = 350;
+          printMenu.printTimer.repeat = false;
+          printMenu.printTimer.triggered.connect(function () {
```

Comment:
Just made a quick test on mobile. On selecting "print to pdf" the first time it sends the open intent one time, on selecting it the second time, it opens two times, on selecting it the third time, three times and so on. Could it be that it conects the signal multiple times?

Reply:
@signedav , fixed.